README.md

```diff
@@ -79,6 +79,10 @@ Provides solutions for precision farming, enabling farmers to monitor their crop
 
   Offers a biomaterial produced from food and vegetable waste, that can be used for packaging as an alternative to plastic. This material is translucent, resistent, and home compostable.
 
+- [Chakr Innovation](https://chakr.in) | India | 2015
+
+  Developed world’s first retro-fit emission control device for diesel generators. This technology can capture ~90% of particulate matter emissions from the exhaust of diesel generators without causing any adverse impact on the engine, and convert it into non-toxic paints and inks.
```

Comment:
"world's first" sounds a bit like a sales pitch, I would just say "Developed a retro-fit..."

Reply:
right, got it